### PR TITLE
Add a new line on SpaceRoomView

### PR DESCRIPTION
**Pull request by luixxiul**
This commit splits the description on SpaceRoomView.tsx into two lines.

Closes https://github.com/vector-im/element-web/issues/21325

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

After: 
after

Notes: none
element-web notes: none

---
This PR currently has no changelog labels, so will not be included in changelogs.

A reviewer can add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is, or add `Type: [enhancement/defect/task]` to the description and I'll add them for you.

Preview: https://pr8002--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Review by t3chguy — CHANGES_REQUESTED** [review]
`description` wasn't necessarily just a string, you could have passed the `<br />` from the caller side, would be much cleaner IMO.

**Comment by luixxiul** [comment]
I'm not completely sure if I got your request correctly, but I force-pushed a new commit.

File: src/components/structures/SpaceRoomView.tsx [hunk 1]
```diff
@@ -897,10 +897,12 @@ export default class SpaceRoomView extends React.PureComponent<IProps, IState> {
...
-                    description={
-                        _t("Let's create a room for each of them.") + "\n" +
-                        _t("You can add more later too, including already existing ones.")
-                    }
+                    description={_t(
+                        "Let's create a room for each of them.<br />" +
+                        "You can add more later too, including already existing ones.",
+                        {},
+                        { 'br': <br /> },
+                    )}
```
Comment (t3chguy) [inline src/components/structures/SpaceRoomView.tsx:905]: How about something more like 

```typescript
description={<>
  { _t("Let's create a room for each of them.") }
  <br />
  { _t("You can add more later too, including already existing ones.") }
</>}
```

Comment (luixxiul) [reply src/components/structures/SpaceRoomView.tsx:905]: Thank you. Obviously I did not know what I was doing :sweat_smile:

**Comment by luixxiul** [comment]
Thanks for the advice. I updated the PR following the suggestion.

**Review by t3chguy — APPROVED** [review]
Great! Thanks for this

**Pull request merged**